third_party/2and3/sqlalchemy/engine/base.pyi

```diff
@@ -1,20 +1,28 @@
-from typing import Any, List, Tuple
+from typing import Any, Iterable, Iterator, List, Mapping, Optional, Tuple
 
 # Dummy until I figure out something better.
 class Connectable:
```

Comment:
So what is your suggestion here? Add all the [`Connectable`](http://docs.sqlalchemy.org/en/latest/core/connections.html#sqlalchemy.engine.Connectable) methods and move it into `interface.pyi`?